Jenkinsfile

```diff
@@ -93,13 +93,14 @@ node('test') {
                         currentBuild.result = 'FAILURE'
                     }
 
-                    junit 'target/junit/ci*/**.xml'
+                    junit 'target/junit/TEST-UI Functional Tests.xml'
                 }
             }
 
         }     
     } catch (e) {
         echo 'This will run only if failed'
+        junit 'target/junit/TEST-UI Functional Tests.xml'
```

Comment:
Not sure why we we need this line here?

Reply:
Hi Ella, this is borrowed from bfs 7.7 development experience I think, when exception happened during test crash, the junit test report is not generated as expected.  And In test all pass scenario, this won't Tigger.

Reply:
What Tengda said is correct. When not in the catch block the report is not created and results in the same error.

Reply:
Do you know why Integration test has a * in the end `junit 'target/junit/TEST-Jest Integration Tests*.xml'`.  Unit test has it as well.
Do we need it for functional test?

Reply:
I don't believe so as the previous version of the functional test junit script was pulled from a jenkinsfile that had that same syntax you point out and in our environment it did not work. That being said I will run a test to see if there is an advantage to using it before I merge this PR through.

Reply:
okay

Reply:
- Looking back at the reports on [7.7.1](https://jenkins.bfs.sichend.people.aws.dev/job/Kibana/job/bfs7.7.1/173/execution/node/3/ws/target/junit/) I believe the `*` is allowing multiple report files/folders to be created. 
- The [results](https://jenkins.bfs.sichend.people.aws.dev/blue/organizations/jenkins/Kibana/detail/bfs6.7.2_test/146/pipeline) of testing this are that the previous error occurs again.
- If you look at Job 144 mentioned in the initial PR description it shows it working as expected.